js/core/utils/style.js

```diff
@@ -111,8 +111,30 @@ const setHeight = function(elements, value) {
     setDimensionProperty(elements, 'height', value);
 };
 
-const setStyle = function(element, value) {
-    element.style.cssText = value;
+// NOTE: Setting the 'element.style' requires creating attributeNode when both of the conditions met:
+//       - a form contains an input with the name property set to "style";
+//       - a form contains a dx-validator (or other popup widget).
+//       T941581
+const setStyle = function(element, styleString, resetStyle = true) {
+    if(!domAdapter.isNode(element.style)) {
+        if(resetStyle) {
+            const styleList = [].slice.call(element.style);
+            styleList.forEach((propertyName) => {
+                element.style.removeProperty(propertyName);
+            });
```

Comment:
Use `element.style.cssText = ''` to remove all styles

Reply:
https://w3c.github.io/webappsec-csp/#directive-style-src
by the csp docs csstext should also throw an error, despite it is implemented not everywhere